src/user-script-obj.js

```diff
@@ -249,7 +249,11 @@ window.EditableUserScript = class EditableUserScript
     // of the generated parts.
     this._evalContent
         // Note intentional lack of line breaks before the script content.
-        = `try { (function scopeWrapper(){ function userScript() { ${this._content} }
+        = `try {
+        (function scopeWrapper(){
+        function userScript() {
+        ${this._content}
+        /* catch open comment blocks */ }
```

Comment:
A) I prefer full/grammatical sentences in comments
B) Something like `/* Force a line break after `_content` in case its final line contains a line comment. /*` seems much more useful; just "catch open comment blocks" is so terse as to be hard to understand later.

Reply:
Alright. Will adjust.